src/renderers/server-render.jsx

```diff
@@ -124,6 +118,15 @@ const getRouterRenderer = (
 	};
 };
 
+const makeRenderer$ = (config: Object) => makeRenderer(
+		config.routes,
+		config.reducer,
+		config.clientFilename,
+		config.assetPublicPath,
+		config.middleware,
+		config.baseUrl
+	);
```

Comment:
I'll address this later (but before closing the pr). At the moment, flow looks like gibberish to me.